readthedocs/core/views/hooks.py

```diff
@@ -75,9 +92,17 @@ def build_branches(project, branch_list):
         not_building - a list of branches that we won't build
     """
     for branch in branch_list:
-        versions = project.versions_from_branch_name(branch)
+        commit = branch['last_commit']
+        versions = project.versions_from_branch_name(branch['name'])
```

Comment:
Why did you change `branch` by `branch['name']` here?

Reply:
See https://github.com/rtfd/readthedocs.org/pull/3457#discussion_r158978235